### ISSUE TITLE
load_app() fails on R 3.2.0

### ISSUE THREAD

**Issue by jranke** [jverzani/gWidgetsWWW2#24]
saying:

```
Error in cur_port > 0 : 
  comparison (6) is possible only for atomic and list types
```

I could track the problem to this code fragment

```{r}
env <- environment(tools::startDynamicHelp)
cur_port <- get("httpdPort", env)
```

in 

https://github.com/jverzani/gWidgetsWWW2/blob/master/R/load.R#L32-33

This returns in R 3.1.2

```
> cur_port
[1] 0
```

and in R 3.2.0

```
> cur_port
function (new) 
{
    if (!missing(new)) 
        port <<- new
    else port
}
<bytecode: 0x...>
<environment: 0x...>
```

**Comment by jverzani** [comment]
Hopefully, this is addressed with beb5ef9ad8c51fba455bfe0cab08444ff6a0376a

I'm not sure how backwards compatible this is (e.g., R3.1.2), but I think it should be. Don't have a good way to test.

Thanks,

**Comment by jranke** [comment]
@jverzani: Thanks a lot for your quick fix for this! I tagged a release and synced it to R-Forge, where the package is now available and working fine under R 3.2.0. I also tested it under R 3.1.2 on Windows, this is also OK. In addition, R 3.1.2 under Windows does not install packages from 
 http://r-forge.r-project.org/bin/windows/contrib/3.2
where the new package is located.
